blueocean-rest/src/main/java/io/jenkins/blueocean/rest/pageable/PagedResponse.java

```diff
@@ -29,7 +30,7 @@
 @Target(METHOD)
 @InterceptorAnnotation(PagedResponse.Processor.class)
 public @interface PagedResponse {
-    public static final int DEFAULT_LIMIT=100;
+    int DEFAULT_LIMIT=100;
```

Comment:
Does this make this member modifiable?  Why change?

Reply:
`public static final` for an interface it's just redundant